LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-valid.html

```diff
@@ -15,7 +15,7 @@
 test_valid_value("column-rule", "dotted");
 test_valid_value("column-rule", "red");
 
-test_valid_value("column-rule", "currentcolor hidden medium", "medium hidden currentcolor");
+test_valid_value("column-rule", "currentcolor hidden medium", "hidden");
```

Comment:
OK.

Reply:
I studied this more, and I realized that the issue you filed, https://github.com/w3c/csswg-drafts/issues/8155, is about the ambiguity when there are multiple minimal representations, like if `medium none currentcolor` is specified it could serialize as `currentcolor`, `none`, or `medium`. This case is not like that, since `hidden` is not the initial value for `column-rule-style`. So there is no other single-word representation. Besides `hidden` we could use `medium hidden` or `medium currentcolor` or `hidden currentcolor` or `medium hidden currentcolor`.

Do you still think I should use an array and keep the old expectation? If so, why? Maybe we should not have web platform tests require minimal representations. Some tests do require that, presumably cases where some other engine was already correctly doing a minimal representation, but not WebKit yet. If there are some cases where a non-minimal representation is needed for web compatibility, then we would definitely not want to require the minimal one. But I really don’t know how to decide when to be strict and when to be lenient in WPT.

Reply:
@Loirooriol I am going to wait on your answer before merging this in to WebKit or WPT; for now I am not making any change though.

Reply:
Ah, I just associated `hidden` with `none` and thought it was the initial value.
But I guess the change is correct according to the shortest serialization principle.